packages/botonic-react/src/index.d.ts

```diff
@@ -198,11 +201,37 @@ export class WebchatApp {
   updateWebchatSettings(settings: WebchatSettingsProps): void
 }
 
+export interface WebChatContextProps {
```

Comment:
I think `setReplies` is missing

Reply:
It's still missing, that's because it's not needed?

Reply:
`WebChat` --> `Webchat`

Reply:
> It's still missing, that's because it's not needed?

I forgot to mention, it's deprecated in favour of updateReplies

Reply:
> `WebChat` --> `Webchat`

fixed, thanks